src/compressed_tensors/quantization/lifecycle/forward.py

```diff
@@ -312,6 +313,10 @@ def wrapped_forward(self, *args, **kwargs):
     setattr(module, "forward", bound_wrapped_forward)
 
 
+def unwrap_module_forward_quantized(module: Module):
+    delattr(module, "forward")  # revert to class implementation
```

Comment:
does this then expose the forward method on the `Linear` or `CompressedLinear` class? why do we want to delete the attr?

Reply:
Yes, this exposes the forward method of the class implementation